CoolAbhi1290 Periodical Branch Merge

tresabhi opened the pull request.

Commit:
Merge branch 'master' into CoolAbhi1290

Commit:
Bee Nest and Bee Hive

Commit:
Bamboo

Commit:
Anvil

Commit:
Merge branch 'master' into CoolAbhi1290

Commit:
Cactus

Commit:
Mushrooms

Commit:
Honey

Omg, my ai worked lol. These textures are all ai generated! OMG LOL!!!!!!!!

Commit:
Campfires

These are also ai generated

Commit:
Command Blocks 2

AI ROCKS

Commit:
Hoppers

Powered by AI. I really need to name this ai boy

Commit:
End Portal Frame

Commit:
Nether Portal

Commit:
Prismarine

Commit:
Pumpkin Top

Commit:
Scaffolding

Commit:
Slime

Commit:
Sweet Berries

Commit:
Turtle Eggs

Commit:
Sea Grass (AI)

Commit:
Sea Pickles (Assisted AI)

Commit:
Lava (AI)

Commit:
Dried Kelp (AI)

Commit:
Cake (Assisted AI)

Commit:
End Rod (AI)

Commit:
Fire (AI)

Commit:
Wheat (AI)

Commit:
Lentern, Magma, Vine, White Stained Glass Pan Top, White Tulip, Yellow Stained Glass Pane Top (AI)

Commit:
Tall Grass Bottom, Tall Grass Top, Tnt Bottom, Tnt Side, Torch, Tripwire, Tripwire Hook (AI)

Commit:
Purple Stained Glass Pane Top, Red Stained Glass Pane Top, Red Tulip, Redstone Torch, Redstone Torch Off, Rose Bush Bottom, Rose Bush Top, Spawner, Spruce Door Botom, Spruce Door Top, Spruce Trapdoor, Sunflower Back, Sunflower Bottom, Sunflower Front, Sunflower Top (AI)

Commit:
Observer, Glass Panes, Flowers, Pistons, Potatoes (AI)

Commit:
Jigsaw Blocks, Jungle Related Blocks, Ferns, Glass Panes, etc. (AI)

Commit:
Jigsaw Block Fix (M)

Commit:
Jukebox Fix (M)

Commit:
Lever Fix (M)

Commit:
Glass Pane Fix (M)

Commit:
Lilac Fix (M)

Commit:
Lily Pad Fix (M)

Commit:
Nether Wart Iteration (M)

Commit:
Attached Pumpkin Init (M)

Commit:
Redstone Torch Fix (M)

Commit:
Rose Bush Iteration (M)

Commit:
Spawner Fix (M)

Commit:
Sunflower Fix (M)

Commit:
Torch Fix (M)

Commit:
Tripwire Fix (M)

Commit:
Lantern Fix (M)

Commit:
End Rod Fix (M)

Commit:
Dired Kelp Block Iteration (M)

Commit:
Turtle Egg Fix (M)

Commit:
Sweet Berry Bush Iteration (M)

Commit:
Slime Block Fix (M)

Commit:
Scaffolding Fix (M)

Commit:
End Portal Frame Fix (M)

Commit:
Flower Pot (M)

Commit:
Iron Bar Iteration (M)

Commit:
Item Frame (M)

Commit:
Fern (M)

Commit:
Farm Land Iteration (M)

Commit:
Enchanting Table Iteration (M)

Commit:
Ender Dragon Egg (M)

Commit:
Destroy Stages (AI)

Commit:
Dead Bush (Assisted AI)

Commit:
Dead Corals (AI)

Commit:
Daylight Sensor (AI)

Commit:
Crafting Table (AI)

Commit:
Dark Oak Trap Door (AI)

Commit:
Composter (AI)

Commit:
Cocoa (M)

Commit:
Quarks Block (AI)

Commit:
Cobweb (AI)

Commit:
Carrots Stages (AI)

Commit:
Acacia Door Iteration (Assisted AI)

Commit:
Acacia Trapdoor (AI)

Commit:
Allium (M)

Commit:
Melon Stem (M)

Commit:
Beetroots Stages (AI)

Commit:
Birch Door (AI)

Commit:
Birch Trapdoor (AI)

Commit:
Bubble Coral Block (AI)

Commit:
CTM Compatibility (Read More)

Added the ctm template files
Added ct for a few blocks uing ctmc of ctm
Added blocks that need ct

HungieSpoon committed:
tweaked command blocks

HungieSpoon committed:
item frame tweak

HungieSpoon reviewed (dismissed):
make slime block transparent and destroy stages more visible

Commit:
Slime Block Fix (M)

Commit:
Merge branch 'CoolAbhi1290' of https://github.com/ProcrastinationNation/Procrasti8x8 into CoolAbhi1290

Commit:
Destroy Stages Fix (M)

Commit:
Destroy Stages Fix 2 (M)

tresabhi commented:
> make slime block transparent and destroy stages more visible

Done @HungieSpoon, please review and merge.

Commit:
Big Smoke (AI)

Commit:
Drip (M)

Commit:
Effects (M)

Commit:
Explosion (AI)

Commit:
Generic (M)

Commit:
Glitter (M)

Commit:
Particles (AI)

Commit:
SGA Text (Read Me!!!)(AI)

The text was made with cubic method. They are translucent, but minecraft doesn't support translucensy, and does 0, or 1. So, it still works.

Commit:
Spark (README)(AI)

Even though some of them are transparant, minecraft will render them opaque.

Commit:
Spell (README)(AI)

Even though some of them are transparant, minecraft will render them opaque.

Commit:
Diamond Armour Tweaks (M)

Commit:
Nether Star (M)

Commit:
Oak Door Fix (M)

The old one was trash tbh

Commit:
Painting (Assisted AI)

Commit:
Minecart (M)

Commit:
Lingering Bottle (M)

Commit:
Tnt Minecart (M)

Commit:
Structure Void (M)

Commit:
Beetroot

Commit:
Update birch_door.png

Commit:
Update broken_elytra.png

Commit:
Update campfire.png

Commit:
Update carrot.png

Commit:
Update carrot_on_a_stick.png

Commit:
Update chest_minecart.png

Commit:
Update command_block_minecart.png

Commit:
Update comparator.png

Commit:
Update crossbow_arrow.png

Commit:
Update crossbow_firework.png

Commit:
Update crossbow_pulling_0.png

Commit:
Update crossbow_pulling_1.png

Commit:
Update crossbow_pulling_2.png

Commit:
Update crossbow_standby.png

Commit:
Update dark_oak_door.png

Commit:
Update dragon_breath.png

Commit:
Update elytra.png

Commit:
Update empty_armor_slot_boots.png

Commit:
Update empty_armor_slot_chestplate.png

Commit:
Update empty_armor_slot_helmet.png

Commit:
Update empty_armor_slot_leggings.png

Commit:
Update empty_armor_slot_shield.png

Commit:
Update end_crystal.png

Commit:
Update ender_eye.png

Commit:
Update ender_pearl.png

Commit:
Update experience_bottle.png

Commit:
Update fishing_rod.png

Commit:
Update fishing_rod_cast.png

Commit:
Update flint_and_steel.png

Commit:
Update flower_pot.png

Commit:
Update furnace_minecart.png

Commit:
Update glass_bottle.png

Commit:
Update gold_chestplate.png

Commit:
Update golden_carrot.png

Commit:
Update honey_bottle.png

Commit:
Update hopper.png

Commit:
Update hopper_minecart.png

Commit:
Update iron_door.png

Commit:
Update item_frame.png

Commit:
Update jungle_door.png

Commit:
Update lantern.png

Commit:
Update lingering_potion.png

Commit:
Update poisonous_potato.png

Commit:
Update potato.png

Commit:
Update potion.png

Commit:
Update potion_overlay.png

Commit:
Update repeater.png

Commit:
Update ruby.png

Commit:
Update spruce_door.png

Commit:
Update spruce_boat.png

Commit:
Update splash_potion.png

Commit:
Update seagrass.png

Commit:
Update sea_pickle.png

Commit:
Update saddle.png

HungieSpoon committed:
revert potions to first texture

couldn't find the actual commit to revert

Commit:
REVERT

Commit:
REVERT

Commit:
Deleted Font Folder (README)

we planned to not change the fonts. So I removed the unnesesary font folder.

Commit:
Glass CTM Init

Commit:
Glass Ctm Ready

Commit:
Full Glass CTM Compatibility

HungieSpoon committed:
Merge branch 'CoolAbhi1290' of https://github.com/ProcrastinationNation/Procrasti8x8 into CoolAbhi1290

HungieSpoon committed:
Revert "Update dragon_breath.png"

This reverts commit 176645ee0823d47b9cfc6f6712d0220c42ef6f33.

Commit:
Diamond Block CTM Revamp (README)

The old one looked trash, so I remade it!

Commit:
Merge branch 'CoolAbhi1290' of https://github.com/ProcrastinationNation/Procrasti8x8 into CoolAbhi1290

HungieSpoon committed:
Update acacia_door.png

Commit:
Emerald Block CTM Redo (README)

The old one was trash, so i made a new one

HungieSpoon committed:
Merge branch 'CoolAbhi1290' of https://github.com/ProcrastinationNation/Procrasti8x8 into CoolAbhi1290

HungieSpoon committed:
Revert "Update flint_and_steel.png"

This reverts commit b7f2c320b011a3a10437c36f68ea0e2150b47bd8.

HungieSpoon committed:
Revert "Update ender_pearl.png"

This reverts commit d69c275fd3421d15c2cad91bd918d9933ca4060b.

HungieSpoon committed:
Revert "Update ender_eye.png"

This reverts commit eba0be44f39a44024fe07d93284e8a9bffe39989.

Commit:
Gold Block CTM Redo

HungieSpoon committed:
Revert "Update end_crystal.png"

This reverts commit 81d490283dfd4ba953ccce272bbc255671c26f73.

Commit:
Lapis Block CTM Redo

HungieSpoon committed:
Update item_frame.png

Commit:
Fixed Weird Diamond Block CTM Highlight Issues

Commit:
Merge branch 'CoolAbhi1290' of https://github.com/ProcrastinationNation/Procrasti8x8 into CoolAbhi1290

Commit:
Fixed Emerald Block CYM Highlights

Commit:
Added All Sandstone Types CTM Compatibility

HungieSpoon committed:
tweaked destroy stages

HungieSpoon committed:
Merge branch 'CoolAbhi1290' of https://github.com/ProcrastinationNation/Procrasti8x8 into CoolAbhi1290

Commit:
Removed CTM Uncamptible textures

Commit:
Merge branch 'CoolAbhi1290' of https://github.com/ProcrastinationNation/Procrasti8x8 into CoolAbhi1290

Commit:
Glass CTM Compatibility

Commit:
Lentern Fix
